src/stk500v2.c

```diff
@@ -2080,6 +2099,8 @@ static int stk500v2_jtag3_parseextparms(const PROGRAMMER *pgm, const LISTID extp
         msg_error("  -xvtarg               Read on-board target supply voltage\n");
         msg_error("  -xvtarg=<arg>         Set on-board target supply voltage\n");
       }
+      if(str_starts(pgmid, "pickit4") || str_starts(pgmid, "snap"))
+        msg_error("  -xmode=avr|pic        Set programmer to AVR or PIC mode\n");
```

Comment:
add comma then exit

Reply:
Comma? like `Set programmer to AVR, or PIC mode\n`?

It already exits after printing -xhelp:

```c
      if(str_starts(pgmid, "pickit4") || str_starts(pgmid, "snap"))
        msg_error("  -xmode=avr|pic        Set programmer to AVR or PIC mode\n");
      msg_error  ("  -xhelp                Show this help menu and exit\n");
      exit(0);
```

Reply:
Nevermind. I started looking at the review from the bottom of this page. I'll assume this is what you wanted:

```c
msg_error("  -xmode=avr|pic        Set programmer to AVR or PIC mode, then exit\n");
```